### PR TITLE
Immersive article bugs

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Makes sure that the headline is displaying when there is no image, and fixes some issues with pull quote styles in immersive and photo essay. 

@regiskuckaertz @NataliaLKB @GHaberis

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss [hunk 1]
```diff
@@ -714,14 +740,23 @@
...
+        height: 105px;
 
         .drop-cap__inner {
             font-weight: 900;
-            font-size: 9.5em;
+            font-size: 140px;
             line-height: .7; // To optically align to top of five lines of text
+
+            @include mq($from: tablet) {
+                font-size: 138px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_article-immersive.scss]: I tried the promising [`initial-letter`](https://developer.mozilla.org/en-US/docs/Web/CSS/initial-letter) but alas it is not available in many browsers yet 😩

**Review by regiskuckaertz — APPROVED** [review]
looks good

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss [hunk 2]
```diff
@@ -380,7 +261,7 @@
 
     .element-pullquote.element--showcase {
         @include mq(tablet) {
-            margin-bottom: $gs-baseline * 3;
+            // margin-bottom: $gs-baseline * 3;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_article-immersive.scss]: Can we delete this, not just comment it out?

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss [hunk 3]
```diff
@@ -645,14 +647,37 @@
...
         padding: 0;
         padding-bottom: $gs-baseline * 2;
         font-weight: 700;
 
         @include mq(desktop) {
             font-size: 50px;
         }
+
+        &:before {
+            position: absolute;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_article-immersive.scss]: Can we add a comment explaining what this is? Its just nice when we are reading the code in the feature :)

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss (hunk 2 above)
Comment (blongden73) [reply static/src/stylesheets/module/content-garnett/_article-immersive.scss]: Done.

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss (hunk 3 above)
Comment (blongden73) [reply static/src/stylesheets/module/content-garnett/_article-immersive.scss]: done.

**Review by NataliaLKB — APPROVED** [review]
Great! 💯

**Pull request merged**